wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/subsystems/Claw.java

```diff
@@ -30,18 +33,19 @@ public void log() {
   }
 
   /** Set the claw motor to move in the open direction. */
-  public void open() {
-    m_motor.set(-1);
+  public Command open() {
+    return runOnce(() -> m_motor.set(-1))
+        .andThen(Commands.waitSeconds(1))
+        .andThen(m_motor::stopMotor)
+        .withName("open claw");
   }
 
   /** Set the claw motor to move in the close direction. */
-  public void close() {
-    m_motor.set(1);
-  }
-
-  /** Stops the claw motor from moving. */
-  public void stop() {
-    m_motor.set(0);
+  public Command close() {
+    return runOnce(() -> m_motor.set(1))
+        .andThen(Commands.waitSeconds(1))
+        .andThen(m_motor::stopMotor)
```

Comment:
This doesn't seem correct (or at least, a direct translation)- The former implementation of [`CloseClaw`](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/commands/CloseClaw.java) uses `m_claw.isGrabbing()` as the condition for `isFinished()`, and `end(boolean interrupted)` checks `RobotBase.isSimulation()`.